### PR TITLE
PLA-6250 design workflow cleanup

### PR DESCRIPTION
**Pull request by talltea**
# Citrine Python PR

## Description 
Minor clean up after https://github.com/CitrineInformatics/citrine-python/pull/502 

### PR Type:
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

### Adherence to team decisions
- [ ] I have added tests for 100% coverage
- [ ] I have written Numpy-style docstrings for every method and class.
- [ ] I have communicated the downstream consequences of the PR to others.
- [ ] I have bumped the version in setup.py

### CODE REVIEW

File: src/citrine/_rest/pageable.py [hunk 1]
```diff
@@ -46,6 +54,8 @@ def _fetch_page(self,
             {'json':
                 {'search_params': {'name': {'value': 'Project', 'search_method': 'SUBSTRING'}}}
             }
+        additional+params: dict, optional
```
Comment (maxhutch) [inline src/citrine/_rest/pageable.py:57]:
```suggestion
        additional_params: dict, optional
```

Comment (bfolie) [inline src/citrine/_rest/pageable.py:57]:
```suggestion
        additional_params: dict, optional
```

File: src/citrine/resources/design_execution.py [hunk 2]
```diff
@@ -68,7 +75,6 @@ def _build_candidates(cls, subset_collection: Iterable[dict]) -> Iterable[Design
         for candidate in subset_collection:
             yield DesignCandidate.build(candidate)
 
-    @lru_cache()
```
Comment (maxhutch) [inline src/citrine/resources/design_execution.py:71]: Why no lru?

File: src/citrine/resources/design_execution.py [hunk 3]
```diff
@@ -48,6 +48,13 @@ class DesignExecution(Resource['DesignExecution'], Pageable):
         'experimental_reasons',
         serializable=False
     )
+    archived = properties.Boolean('archived', default=False)
```
Comment (maxhutch) [inline src/citrine/resources/design_execution.py:51]: I don't think we need the default anymore.

**Review by maxhutch — COMMENTED** [review]
A few comments inline.  Are you going to deprecate the old Workflow and Execution stuff in a separate PR?

File: src/citrine/resources/design_execution.py (hunk 2 above)
Comment (talltea) [reply src/citrine/resources/design_execution.py:71]: https://citrine.atlassian.net/browse/PLA-6294

**Comment by talltea** [comment]
> A few comments inline. Are you going to deprecate the old Workflow and Execution stuff in a separate PR?

yeah, that's the plan. i want to update the docs and add a small e2e test before hitting the deprecation button.

Note: build failing because of version bump

**Review by bfolie — APPROVED** [review]

**Review by maxhutch — DISMISSED** [review]

**Review by bfolie — APPROVED** [review]

**Pull request merged**